parts/linux/cloud-init/artifacts/mariner/cse_install_mariner.sh

```diff
@@ -25,6 +25,17 @@ installDeps() {
     fi
 }
 
+installKataDeps() {
+    if [[ $OS_VERSION != "1.0" ]]; then
+      # TODO - remove kernel-uvm-devel once we build the vanilla Kata UVM in our UVM build pipeline
```

Comment:
We should be able to do this now. And if so, we should remove the comment and kernel-uvm-devel from the list